src/layers/legacy/medImageIO/itkGISImageIO.cpp

```diff
@@ -18,6 +18,11 @@
 
 #include <fstream>
 
+#ifdef WIN32
+#define snprintf sprintf_s
+//#define strncpy(x, y, z) strcpy_s(x, z, y)
```

Comment:
there is a commented line which can be removed

Reply:
Ok